lib/models/actions.js

```diff
@@ -62,6 +63,9 @@ function conditionalExec(action, event, lastTime, callbackW) {
                 case 'post':
                     postAction.doIt(action, event, callbackS);
                     break;
+                case 'twitter':
```

Comment:
Yes , I thought about it , but i decided to wait for next action. They are only a few yet